docs/site/themes/template/layouts/partials/getting-started.html

```diff
@@ -8,7 +7,7 @@ <h2>Join us!</h2>
 		{{ else }}
 			<h2>Ready to dive in?</h2>
 			<p>Our documentation is a great place to start!</p>
-			<a href="/docs/" class="button navy-blue">Documentation</a>
+			<a href="/docs/{{ $.Site.Params.docs_latest }}" class="button navy-blue">Documentation</a>
```

Comment:
I don't think we want this change. This will cause docs from past releases to be redirected to the latest release.